docs/dev.md

```diff
@@ -9,7 +9,7 @@ about the basemap data.
 
 ### Prerequisites
 There are a couple of requirements for building and running a new Atlas Check. The following are required:
-- [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html) 
+- [Java 11](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html) 
```

Comment:
Thanks for updating this! Please use: https://adoptopenjdk.net/?variant=openjdk11&jvmVariant=hotspot